docs/non_containerized_NSFS.md

```diff
@@ -395,31 +395,35 @@ These error codes will get attached with a specific Bucket or Account schema ins
 Users can create, update, delete, and list buckets and accounts using CLI. If the config directory is missing CLI will create one and also create accounts and buckets sub-directories in it and default config directory is `${config.NSFS_NC_DEFAULT_CONF_DIR}`. 
 
 CLI will never create or delete a bucket directory for the user if a bucket directory is missing CLI will return with error.
+
+
```

Comment:
can you please also change the NSFS_SERVICE_FAILED on health error codes to the new error code please?